<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionClose_Project"/>
    <addaction name="actionClose_project_without_saving"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionOpen_text"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_text"/>
    <addaction name="actionImport_FlexText"/>
    <addaction name="actionImport_plain_text"/>
    <addaction name="separator"/>
    <addaction name="actionMerge_translations_from_other_FlexText_file"/>
   </widget>
   <widget class="QMenu" name="menuGuts">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionRemove_unused_gloss_items"/>
    <addaction name="actionPerform_a_query"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionWriting_systems"/>
    <addaction name="actionDelete_text"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionSearch_gloss_items"/>
    <addaction name="actionSubstring_search_gloss_items"/>
    <addaction name="actionRaw_XQuery"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuProject"/>
   <addaction name="menuSearch"/>
   <addaction name="menuGuts"/>
  </widget>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="text">
    <string>Save Project As..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAdd_text">
   <property name="text">
    <string>Paste in text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionImport_FlexText">
   <property name="text">
    <string>Import FlexText...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionOpen_text">
   <property name="text">
    <string>Open text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionImport_plain_text">
   <property name="text">
    <string>Import plain text...</string>
   </property>
  </action>
  <action name="actionWriting_systems">
   <property name="text">
    <string>Writing systems...</string>
   </property>
  </action>
  <action name="actionDelete_text">
   <property name="text">
    <string>Delete text...</string>
   </property>
  </action>
  <action name="actionClose_project_without_saving">
   <property name="text">
    <string>Close project without saving</string>
   </property>
  </action>
  <action name="actionMerge_translations_from_other_FlexText_file">
   <property name="text">
    <string>Merge translations from other FlexText file...</string>
   </property>
  </action>
  <action name="actionSearch_gloss_items">
   <property name="text">
    <string>Search gloss items...</string>
   </property>
  </action>
  <action name="actionFor_Text_Form">
   <property name="text">
    <string>For Text Form...</string>
   </property>
  </action>
  <action name="actionSubstring_search_gloss_items">
   <property name="text">
    <string>Substring-search gloss items...</string>
   </property>
  </action>
  <action name="actionRaw_XQuery">
   <property name="text">
    <string>Raw XQuery...</string>
   </property>
  </action>
  <action name="actionRemove_unused_gloss_items">
   <property name="text">
    <string>Remove unused gloss items</string>
   </property>
  </action>
  <action name="actionPerform_a_query">
   <property name="text">
    <string>Perform a query...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
